BuyerRatio Dataset

In [1]:
import pandas as pd
import numpy as np
from scipy import stats

# Problem Statement :

Sales of products in four different regions is tabulated for males and females. Find if male-female buyer rations are similar across regions

# 1 - Business Problem

Is the male-female buyer rations are similar across regions

# 2 - Data description

α == 0.05 (95% Confidence)

Y == Discrete
X == Discrete

Since there are more than 2 variable we will perform Chi-Square test

# 3 - Chi-Square Test

In [3]:
df = pd.read_csv("C:/Users/Prabinkp/Desktop/ASSIGNEMNETS_DS/BuyerRatio.csv")

H0 == The male-female buyer rations are similar across regions


H1 == The male-female buyer rations are not similar across regions

In [4]:
stats.chi2_contingency([df["East"], df["West"], df["North"], df["South"]])

(1.5959455386610577,
 0.6603094907091882,
 3,
 array([[  42.76531299,  442.23468701],
        [ 146.81287862, 1518.18712138],
        [ 131.11756787, 1355.88243213],
        [  72.30424052,  747.69575948]]))

P value of Chi-Square test == 0.66 > α

# Thus HO is accepted.

# The male-female buyer rations are similar across regions